orchestration/dagster_orchestration/hca_orchestration/resources/__init__.py

```diff
@@ -7,16 +7,16 @@
 
 
 __all__ = [
-    console_slack_client,
-    dataflow_beam_runner,
-    google_storage_client,
-    jade_data_repo_client,
-    live_slack_client,
-    local_beam_runner,
-    local_storage_client,
-    noop_data_repo_client,
-    test_beam_runner,
-    noop_bigquery_client,
-    bigquery_client,
-    load_tag
+    'console_slack_client',
```

Comment:
❓ why quote all of these?

Reply:
i mentioned this briefly in the overview, but it's actually against python spec to pass in the values directly here. you're meant to specify the names of each object being added to `__all__` rather than list the objects themselves.

there's some low-level python explanation for this that i don't fully understand, it has to do with being able to reliably retrieve the objects in question, which may not be possible if you pass their values into `__all__`.

also, it makes mypy sad if you don't make them strings.